core/server/adapters/scheduling/post-scheduling/index.js

```diff
@@ -25,7 +24,8 @@ _private.loadClient = function loadClient() {
 };
 
 _private.loadScheduledPosts = function () {
-    return schedules.getScheduledPosts()
+    const api = require('../../../api');
```

Comment:
Could you elaborate on why require didn't stay on the top? Is it just for application startup speedup reasons? Just wondering if this should be truly decoupled right away and whole `api`  (or just the `api.schedules.getScheduledPosts()` function) could be passed as a parameter to this module somewhere here - https://github.com/TryGhost/Ghost/blob/master/core/server/index.js#L44 ? Or maybe that's the next step :thinking:

Reply:
> Could you elaborate on why require didn't stay on the top? Is it just for application startup speedup reasons? 

Yeah. You should require things when you need it.

> decoupled right away and whole api

Can be, but then we start with dependency injection, which is a big step.